doc/classes/String.xml

```diff
@@ -200,10 +213,25 @@
 				If the string is a valid file path, returns the filename.
 			</description>
 		</method>
+		<method name="get_slice" qualifiers="const">
+			<return type="String" />
+			<argument index="0" name="delimeter" type="String" />
+			<argument index="1" name="slice_index" type="int" />
+			<description>
+				Returns a slice of the string given a [code]delimeter[/code] and a [code]slice_index[/code].
+				For example, [code]print("0|1|2".get_slice("|", 0))[/code] will print [code]0[/code].
```

Comment:
```suggestion
				For example, [code]print("a|b|c".get_slice("|", 1))[/code] will print [code]b[/code].
```
Avoids the confusion between index 0 and the 0-th indexed char which happens to be 0.